src/common/ConfigManager.h

```diff
@@ -133,8 +133,9 @@ extern int videoOption;
 extern int vsync;
 extern int wasPaused;
 extern uint32_t windowHeight;
-extern int windowPositionX;
-extern int windowPositionY;
+extern int windowMaximized;
+extern int windowPositionX, bkpPosX;
+extern int windowPositionY, bkpPosY;
```

Comment:
The `bkpPos{X,Y}` are meant to be one from last position of window. This achieves what I want, but I am not 100% sure if I really need them.